scripts/updateLicense.py

```diff
@@ -70,16 +91,34 @@ def replace(header_lines):
                 f.write(line)
         print(name)
 
+def get_license_type(file):
+    ext_map = {
+        '.go' : LICENSE_BLOB_LINES_GO,
+        '.mk' : LICENSE_BLOB_LINES_MAKEFILE,
+        'Makefile' : LICENSE_BLOB_LINES_MAKEFILE,
+        '.py' : LICENSE_BLOB_LINES_PYTHON,
+        '.sh' : LICENSE_BLOB_LINES_SHELL,
```

Comment:
why do you need the vars, can you not just call `get_license_blob_lines(...)` inline?

Reply:
at minimum you can define local vars, and you only need two of them (all scripts share the same)

Reply:
> why do you need the vars, can you not just call `get_license_blob_lines(...)` inline?

i thought it'd be more readable